internal/controller/taggable_controller.go

```diff
@@ -135,9 +136,14 @@ func (r *TaggableResourceReconciler[T, P, PT]) Reconcile(ctx context.Context, re
 		expectedTagValueRefs = append(expectedTagValueRefs, value.Name)
 	}
 
+	projectInfo, err := r.TagsManager.GetProjectInfo(ctx, projectID)
```

Comment:
`projectID` may be an empty string - e.g. if the annotation is set to one. Any scenario I can think of where this would be the case would also mean the resource is broken on the config connector side to begin with, but do we want to safeguard against this here anyway?

Reply:
Updated to `GetProjectInfo` func to retrurn error if `projectID` is empty